mods/cnc/maps/funpark01/map.yaml

```diff
@@ -0,0 +1,542 @@
+Selectable: False
+
+MapFormat: 6
+
+RequiresMod: cnc
+
+Title: Strange Behavior
+
+Description: There have been some reports of strange animals in this area. \n\nTake your units to investigate, and report back your findings.
+
+Author: Westwood Studios
+
+Tileset: JUNGLE
+
+MapSize: 64,64
+
+Bounds: 10,5,45,52
+
+UseAsShellmap: False
```

Comment:
This needs to become `Visibility: MissionSelector` since https://github.com/openra/openra/pull/7134.